plans/super/177-codex-auth-mode-conflict.md

```
@@ -0,0 +1,600 @@
+
+**Files.**
+- `.claude/rules/centralized-sdk-call.md` — refresh lines ~541-569.
+- `CHANGELOG.md` — new entry (top of unreleased section).
+
+**Depends on.** US-006.
+
+---
+
+### Story map
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)